content/800-data-platform/200-projects/100-add-project/120-import-existing-project-existing-db.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Import an existing project with an existing database'
+metaTitle: 'Import an existing JavaScript/TypeScript project with an existing database in the Prisma Data Platform'
+metaDescription: 'You can import your production application in the Prisma Data Platform and configure the Data Proxy to enable connection pooling and avoid exhausting the database connections in serverless environments.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Import your production application and configure the [Data Proxy](/data-platform/data-proxy) to enable connection pooling and avoid exhausting the database connections in serverless environments.
+
+</TopBlock>
+
+## Prerequisites and considerations
+
+* Log in to your GitHub account ([or sign up for one](https://github.com/signup)).
+* Prepare a GitHub repository with your Node.js application that uses Prisma ORM.
+* Obtain a connection string to a [Prisma ORM-supported database](/reference/database-reference/supported-databases).
+* [Sign up for the Prisma Data Platform](/data-platform/account#sign-up-for-the-prisma-data-platform).
+* [Grant the Prisma GitHub app read and write permissions to your GitHub repositories](/data-platform/projects/add-project#grant-read-and-write-permissions-to-the-prisma-github-app).
+
+## Steps
+
+1. Go to the [_Projects_](https://cloud.prisma.io/projects) page and click **New Project**.
+2. On the _Configure project_ page, name your project and select an existing GitHub repository.
+    1. In **Display Name**, enter a name for your project.
+        <Admonition>
+        
+        **Note**<br /><br />
+        The display name of a project appears in the following places in the Prisma Data Platform:<br />
+          &nbsp;• [_Projects_](https://cloud.prisma.io/projects) page<br />
+          &nbsp;• [_Billing_](https://cloud.prisma.io/billing) page<br />
+          &nbsp;• **Projects** drop-down in the upper left
+        </Admonition>
+    2. From **GitHub Accounts & Organizations**, select a GitHub account or an organization.
+    3. Select **Import a Prisma repository**.
+    4. From **Repository**, select the GitHub repository you want to import.
```

Comment:
> From Repository, select the GitHub repository you want to import. 

You can also connect to more repos if your repo is not visible in the list. Would we want to mention this?